### PR TITLE
[Bugfix] Fix Net tests being omitted

### PR DESCRIPTION
**Pull request by wlan0**
@harshavardhana  It addresses the changes you requested and a bugfix where a condition was excluding all hosts from net tests

### CODE REVIEW

File: cmd/notification.go [hunk 1]
```diff
@@ -911,15 +911,12 @@ func (sys *NotificationSys) NetOBDInfo(ctx context.Context) madmin.ServerNetOBDI
...
-	stripPath := func(hostPath string) string {
-		return strings.Split(hostPath, slashSeparator)[0]
-	}
 
 	hostSet := set.NewStringSet()
 	for _, ez := range globalEndpoints {
 		for _, e := range ez.Endpoints {
-			host := stripPath(e.Host)
-			if hostSet.Contains(host) {
+			host := e.Host
```
Comment (harshavardhana) [inline cmd/notification.go]: you can use e.Host directly now no need to have a new variable @wlan0 
```go
if !hostSet.Contains(e.Host) {
        sortedGlobalEndpoints = append(sortedGlobalEndpoints, e.Host)
        hostSet.Add(e.Host)
}
```

File: cmd/admin-handlers.go [hunk 2]
```diff
@@ -1388,10 +1388,9 @@ func (a adminAPIHandlers) OBDInfoHandler(w http.ResponseWriter, r *http.Request)
 	w.Header().Set(xhttp.ContentType, string(mimeJSON))
 	w.WriteHeader(http.StatusOK)
 
+	enc := json.NewEncoder(w)
 	partialWrite := func() {
-		jsonBytes, _ := json.Marshal(obdInfo)
-		_, err := w.Write(jsonBytes)
-		logger.LogIf(ctx, err)
+		logger.LogIf(ctx, enc.Encode(obdInfo))
```
Comment (harshavardhana) [inline cmd/admin-handlers.go:1393]: Thanks!

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: cmd/notification.go (hunk 1 above)
Comment (wlan0) [reply cmd/notification.go]: Done

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#04a68378-f6cc-4013-aec0-6836e0c43469) |
| mint-gateway-azure.sh | :x: [more...](#3f0e8324-938d-48a3-82a0-ae42f1db17d3) |
<a name="04a68378-f6cc-4013-aec0-6836e0c43469"></a>
### 9234-7e4d0e1/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c4.minio.io:32040
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 750521fa3298:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-6y6iklznurcvap",
    "objectName": "aphunfeizjv0xsvkzc960xwwj4qjyt"
  },
  "duration": 813,
  "error": "RequestError: send request failed\ncaused by: Put http://minio-c4.minio.io:32040/aws-sdk-go-test-6y6iklznurcvap/aphunfeizjv0xsvkzc960xwwj4qjyt?tagging=: EOF",
  "function": "testObjectTagging",
  "message": "AWS SDK Go PUTObjectTagging expected to success but got RequestError",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 15 tests successfully.
```
<a name="3f0e8324-938d-48a3-82a0-ae42f1db17d3"></a>
### 9234-7e4d0e1/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c3.minio.io:31454
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp db1572208aa9:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 8 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 8 minutes and 51 seconds
(4/15) Running aws-sdk-ruby tests ... done in 20 seconds
(5/15) Running awscli tests ... FAILED in 18 seconds
{
  "name": "awscli",
  "duration": 1524,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

**Review by poornas — APPROVED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :x: [more...](#7b0c6e6b-ff2c-4d38-a58c-1ec5d056c45d) |
| mint-gateway-azure.sh | :x: [more...](#01d8eec9-1b44-4368-9b5f-9df1545d27a0) |
<a name="7b0c6e6b-ff2c-4d38-a58c-1ec5d056c45d"></a>
### 9234-59c62de/mint-large-bucket.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-c4.minio.io:31965
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp c57ebdceac8f:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... FAILED in 3 seconds
{
  "alert": "",
  "args": {
    "bucketName": "aws-sdk-go-test-xkjn23rlr3fn9n",
    "objectName": "9do92332rekdfveufbvvfxq42q6pfa"
  },
  "duration": 525,
  "error": "RequestError: send request failed\ncaused by: Put http://minio-c4.minio.io:31965/aws-sdk-go-test-xkjn23rlr3fn9n/9do92332rekdfveufbvvfxq42q6pfa?tagging=: EOF",
  "function": "testObjectTagging",
  "message": "AWS SDK Go PUTObjectTagging expected to success but got RequestError",
  "name": "aws-sdk-go",
  "status": "FAIL"
}

Executed 0 out of 15 tests successfully.
```
<a name="01d8eec9-1b44-4368-9b5f-9df1545d27a0"></a>
### 9234-59c62de/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev3.minio.io:32067
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 46b7e2810243:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 9 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 8 minutes and 38 seconds
(4/15) Running aws-sdk-ruby tests ... done in 19 seconds
(5/15) Running awscli tests ... FAILED in 19 seconds
{
  "name": "awscli",
  "duration": 1499,
  "function": "make_bucket\n",
  "status": "FAIL",
  "error": "An error occurred (BucketAlreadyOwnedByYou) when calling the CreateBucket operation: Your previous request to create the named bucket succeeded and you already own it."
}

Executed 4 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Pull request merged**